src/main/java/de/muenchen/dave/services/SucheService.java

```diff
@@ -182,26 +190,37 @@ public SucheComplexSuggestsDTO complexSuggest(final String query, final boolean
      * @return passende Zaehlstellen
      */
     @Cacheable(value = CachingConfiguration.SUCHE_ZAEHLSTELLE_DATENPORTAL, key = "{#p0, #p1}")
-    public Set<ZaehlstelleKarteDTO> sucheZaehlstelleSichtbarDatenportal(final String query, final boolean noFilter) {
+    public Set<ErhebungsstelleKarteDTO> sucheZaehlstelleSichtbarDatenportal(final String query, final boolean noFilter) {
         log.debug("Zugriff auf den Service #sucheZaehlstelleSichtbarDatenportal");
-        return this.sucheZaehlstelle(query, noFilter)
-                .stream()
-                .filter(zaehlstelleKarte -> ObjectUtils.isEmpty(zaehlstelleKarte.getSichtbarDatenportal())
-                        || zaehlstelleKarte.getSichtbarDatenportal())
-                .collect(Collectors.toSet());
+        final var sichtbarDatenportal = true;
+        return this.sucheErhebungsstelle(query, noFilter, sichtbarDatenportal);
     }
 
     /**
      * Sucht alle freigegebenen Zählstellen und gibt diese an getZaehlstelleKarteDTOS weiter.
```

Comment:
Kommentar an Erhebungsstellen anpassen (Gilt für alle Kommentare in der Klasse)

Reply:
done